src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -2317,4 +2248,73 @@ public void remove(String ownerKey, String contentId) {
         this.contentAccessManager.syncOwnerLastContentUpdate(owner);
     }
 
+    private Owner createOwnerFromDTO(OwnerDTO ownerDTO) {
+        // Verify that we have an owner key (as required)
+        if (StringUtils.isBlank(ownerDTO.getKey())) {
+            throw new BadRequestException(i18n.tr("Owners must be created with a valid key"));
+        }
```

Comment:
This should stay in the resource layer (so in `OwnerResource.createOwner()`), along with the reverse-translations call(`populateEntity`), which creates the Owner object, which is then passed into the service layer (`OwnerManager.createOwner()`)

Reply:
It turns out doing this is has consequences that are hard to fix without a wider refactoring. That is the kind of task that needs to be done as a PoC. The current changes are good enough for sharing the owner creating code between the 2 endpoints.